Releases.md

```diff
@@ -0,0 +1,11 @@
+# Process for Releasing
+
+To create a new release, either push a tag via git, or go to the GitHub [releases tab](https://github.com/justeat/JustSaying/releases) and click `Draft a new release`.
```

Comment:
I think the UI is the only method we should use. I seem to remember that someone made a tag in a repo somewhere recently, and for whatever reason it didn't trigger all the things that make a release work.

Reply:
I know that I have got it wrong a few times, I only just learned you could do it through the UI! I'm a big fan of that.

Reply:
That would have been me not getting it to work property.